src/starknet_runner/runner.rs

```diff
@@ -377,4 +377,124 @@ mod tests {
             expected_call_info
         );
     }
+
+    #[test]
+    fn integration_storage_test() {
+        // ---------------------------------------------------------
+        //  Create program and entry point types for contract class
+        // ---------------------------------------------------------
+
+        let path = PathBuf::from("starknet_programs/storage.json");
+        let contract_class = ContractClass::try_from(path).unwrap();
+        let mut entry_points_by_type = HashMap::new();
+
+        let storage_entrypoint_selector = Felt::from_str_radix(
+            "927d8c7b62a4c3970850ba50fb9bede72a52b22464bcc9303c6142276927f2",
+            16,
+        )
+        .unwrap();
+
+        entry_points_by_type.insert(
```

Comment:
now that we can correctly parse the contract class from the compiled JSON you can remove this. Take @mmsc2 test as an example.